modules/govuk/manifests/apps/mapit.pp

```diff
@@ -117,16 +117,21 @@
         value   => $django_secret_key;
   }
 
-  if ($::mapit_data_present == '0') {
+  if ($::mapit_data_present != '1') {
 
 
     file { '/etc/govuk/import_mapit_data.sh':
-      ensure => file,
-      source => 'puppet:///modules/govuk/etc/govuk/import_mapit_data.sh',
-
+      ensure  => file,
+      mode    => '0755',
+      source  => 'puppet:///modules/govuk/etc/govuk/import_mapit_data.sh',
+      require => Class['govuk_postgresql::server'],
     }
+
     exec { 'populate mapit database':
-      command => '/bin/bash /etc/govuk/import_mapit_data.sh 2>&1 | logger',
+      command     => '/bin/bash /etc/govuk/import_mapit_data.sh 2>&1 | logger',
+      subscribe   => File['/etc/govuk/import_mapit_data.sh'],
```

Comment:
Are you sure you want the subscribe here? i.e. do we want Puppet to re-run the import script if the import script changes? Seems like that would have availability implications, and probably be surprising to someone in future.

Reply:
I will remove that, anyway, it was not going to run because of the conditional below as you rightly saw.